scripts/check_site.lua

```diff
@@ -315,6 +317,20 @@ function need_domain_name(path)
 	end, nil, 'be a valid domain name')
 end
 
+function obsolete(path, msg)
+	local val = loadvar(path)
+	if val == nil then
+		return nil
+	end
+
+	if not msg then
+		msg = 'Please remove or migrate it, check the release notes and documentation for details.'
```

Comment:
Separating these two sentences with a comma looks weird, I'd only use such a construction when the second sentence is very short ("%s is obsolete, please remove or migrate it." was fine IMO). I guess we could just remove the "Please remove or migrate it" completely and make the default msg "Check the release notes and documentation for details.".

Reply:
Removed it. Stashed commit and reword commit msg.